website/docs/components/table/advanced-table/partials/specifications/sticky-column.md

```diff
@@ -0,0 +1,21 @@
+## Sticky headers and columns
+
+The Advanced Table supports setting both the header and the first column as sticky or “pinned” to either the top or left of the container respectively. This can aid in navigation and traversal through large data sets while persisting important identifying values (e.g., names, IDs, etc).
```

Comment:
[suggestion] simplifying and cleaning up some of the repetition. 

```suggestion
The Advanced Table supports setting the header and the first column as sticky or “pinned.” This can help users navigate large data sets while persisting important identifying values, such as names and IDs.
```